ui/src/Groups/Counter/Counter.view.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable react/no-multi-comp */ // disabled as we have a hoc funtion in file
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Body, BODY, Subheading, Text } from '../../Elements/Text/index.js';
+
+const Counter = (props) => {
+  const { title, subtitle, count, countLimit, className, ...others } = props;
+  const classesToApply = `Counter${className ? ` ${className}` : ''}`;
+
+  let countValue = count;
+  if (count < 0) {
+    countValue = '00';
+  } else if (count < 10) {
+    countValue = '0' + count;
+  } else if (count > countLimit) {
+    countValue = countLimit + '+';
+  }
+
+  return (
+    <div {...others} className={classesToApply}>
+      <div className={'Counter--title'}>
```

Comment:
Would these not be elements @nictownsend ? (so `__`) rather than modifiers (`--`)

Reply:
I also didnt see matching classes in the scss for this component come to think of it!